src/Anneal.jl

```diff
@@ -12,32 +12,28 @@ const VI = MOI.VariableIndex
 const CI = MOI.ConstraintIndex
 const Maybe{T} = Union{T, Nothing}
 
+import Test
+
 # -*- Exports: Interface -*-
-export MOI, AbstractSampler, AbstractSamplerAttribute, @anew
+export AbstractSampler, AbstractSamplerAttribute, @anew
 
 # -*- Exports: Submodules -*-
 export ExactSampler, RandomSampler, IdentitySampler
 export SimulatedAnnealer
 
 # -*- Includes: Anneal -*-
-include("error.jl")
-include("tools.jl")
-include("samples.jl")
-include("interface.jl")
-include("MOI_wrapper.jl")
-include("macros.jl")
+include(joinpath("lib", "error.jl"))
+include(joinpath("lib", "tools.jl"))
+include(joinpath("lib", "samples.jl"))
+include(joinpath("interface", "interface.jl"))
+include(joinpath("interface", "MOI_wrapper.jl"))
+include(joinpath("interface", "macros.jl"))
+include(joinpath("interface", "tests.jl"))
 
 # -*- Includes: Submodules -*-
-include("samplers/random/random.jl")
-using .RandomSampler
-
-include("samplers/exact/exact.jl")
-using .ExactSampler
-
-include("samplers/identity/identity.jl")
-using .IdentitySampler
-
-include("annealers/simulated/simulated.jl")
-using .SimulatedAnnealer
+include(joinpath("samplers", "random", "random.jl"))
+include(joinpath("samplers", "exact", "exact.jl"))
+include(joinpath("samplers", "identity", "identity.jl"))
+include(joinpath("samplers", "simulated", "simulated.jl"))
 
 end # module
```

Comment:
Github keeps flagging the missing empty line at the end of your files. I do not know if this is an Issue for Julia, but check it please

Reply:
Opened #19